packages/apps/docs/src/docs/build/pact/beginner/language-basics.md

```diff
@@ -1,205 +1,196 @@
 
-The [Pact Language Reference](/reference) includes an overview of the syntax, basic commands, and built-in functions you’ll use to create smart contracts. By getting familiar with these commands, you’ll be prepared to build simple and safe smart contracts with the Pact programming language.
+For more information about data types, see [Pact syntax](/reference/syntax).
+
+If you aren't sure about the data type when you are working in Pact, you can check its data type by using the Pact `typeof`
+built-in function.
 
-:::
+To try it yourself:
 
-:::info Follow Along Open the online editor at [pact.kadena.io](https://pact.kadena.io/) to follow along with this tutorial. You can run each of the commands described to get more familiar with the Pact programming language.
+1. Open a terminal shell on your local computer.
+2. Start the Pact REPL interpreter you installed in [Install Pact](/build/pact/dev-network#install-pacth1989060372) by running the following command:
+   
+   ```bash
+   pact
+   ```
 
-View the [Pact Online Code Editor Tutorial](/build/pact/dev-network) for more information on running Pact commands. :::
+3. Use the `typeof` built-in function to test different data types.
+   
+   In Pact, functions are enclosed by parenthesis, so to test a list data type, for example:
+   
+   ```pact
+   (typeof [1,2,3])
+   ```
+   
+   After you run the command, the `typeof` function returns that this is a list with any type of data items:
 
-## Pact Language Basics Tutorial
+   ```pact
+   "[<a>]"
+   ```
 
-https://www.youtube.com/watch?v=jILGrrIqP2w
+### String operations
 
-Subscribe to our [YouTube channel](https://www.youtube.com/channel/UCB6-MaxD2hlcGLL70ukHotA) to access the latest Pact tutorials.
+If most cases, you use double quotation marks (" ") around strings to support whitespace or multi-line strings.
+However, you can also represent strings by prepending the string with a single quotation mark (').
 
-## Get Started
+Typically, you use a single quotation mark to identify strings that are used as function names or table names. 
+You can't identify a string with a single quotation mark if the string includes whitespace or requires multiple lines, but this can be a helpful way to identify certain type of strings more succinctly.
 
-To get started with the Pact Language Basics, navigate to the [Pact Smart Contract Language reference](/reference). The Pact Smart Contract Language reference describes the syntax and semantics of the Pact language.
+For more information about using a single quotation mark for function or table names, see [symbols](/reference/syntax).
 
-![1-pact-language-reference](/assets/docs/1-pact-language-reference.png)
+To work with strings:
 
-If you haven’t already, take some time to review this documentation to better understand the Pact language.
+1. Open a terminal shell on your local computer.
```

Comment:
Probably at this point in the tutorial, the reader inferred that `pact>` lines mean they are typing in the REPL, and we can remove further instructions about how to open a terminal and start the REPL.